client/app/dbaas/dbaas-metrics/token/add/metrics-token-add.html

```diff
@@ -1,43 +1,38 @@
-<div class="cui-page cui-page__content">
-    <div class="metrics-token-add__header">
-        <cui-back-link class="metrics-token-add__header-link"
-                    data-state="dbaas.metrics.detail.token"
-                    data-state-params="{ serviceName: MetricsTokenAddCtrl.serviceName }">
-        </cui-back-link>
-        <cui-page-content-title text="'metrics_token_add_token' | translate"></cui-page-content-title>
-    </div>
+<div class="cui-page__content">
```

Comment:
We should only have oui-kit classes. The goal is to have the same visual everywhere.